flytekit/core/constants.py

```diff
@@ -11,6 +11,8 @@
 START_NODE_ID = "start-node"
 END_NODE_ID = "end-node"
 
+DEFAULT_FAILURE_NODE_ID = "nfail"
```

Comment:
should this have eager in its name?

Reply:
The current fail node id is `fn`, Should we use `efn` for eager?

Reply:
Updated to `efn`